### ISSUE TITLE
GUI: Look in the element list for the edit box, activate button, move to button, cancel button. Visually invisible.

### ISSUE THREAD

**Issue by lijianwei2019** [nvaccess/nvda#10655]
### Steps to reproduce:

Open the browser nvda + f7 startup element list, tab to find search box, activate button, move to button, cancel button. The controls are not visible visually.

![image](https://user-images.githubusercontent.com/59390108/71891165-f6897580-3180-11ea-88e1-9d2dbdc2bfa1.png)

### Actual behavior:

Invisible

### Expected behavior:

visible

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:

alpha-19460

#### Windows version:

Windows 10 pro1909

#### Does the issue still occur after restarting your PC?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

no

**Comment by dingpengyu** [comment]
cc
@feerrenrut 
and
@leonardder 
thanks

**Comment by CyrilleB79** [comment]
On my side with Win10 French and NVDA 2019.3beta1 French on this page in Firefox, I an see all the controls of the dialog.

I have also tried to switch NVDA in Chinese Simplified and can also see that the various controls of the dialog appear.

Here are some questions and tracks to help investigate this issue:

* What is the language of Windows?
* What is the language of NVDA's UI
* Does the issue occur on any webpage or on a specific webpage?
* In which browser(s) does the issue occur?
* Does the issue occur with NVDA 2019.2.1?

**Comment by lijianwei2019** [comment]
@CyrilleB79  Hello。
1.Windows language：Simplified Chinese

2.NVDA UI language：Simplified Chinese and English

3.Any webpage

4.IE、Chrome、Firefox 、Edge Will happen

5.This issue will occur in 2019.2.1

My English is bad。Sorry for the language barrier。

Regards.

**Comment by CyrilleB79** [comment]
@lijianwei2019:

* do you have the opportunity to test on another computer?
* When you are on an invisible control tat should be visible could you press NVDA+F1 to get the developper's info for this object and paste this information here?

@ Ohter Chinese people:
Do you experience the same issue (with Windows zh-CN)?

I have tested with the same alpha version (last alhpa) as temporary installer's copy. Unfortunately I could not reproduce this issue. 

From the image posted by @lijianwei2019, I can see something thats seems the focus squeezed at the bottom of the window (with focus highlighting enabled). Yet to be confirmed by someone that have a better vision than me. If provided, dev info on navigator object should confirm this.

Sorry, I do not have any other idea to investigate this issue. Is there anybody who could continue this investigation?

**Comment by lijianwei2019** [comment]
@JulienCochuyt 
Are you interested in solving this problem?

**Comment by JulienCochuyt** [comment]
> @JulienCochuyt
> Are you interested in solving this problem?

I was indeed considering to, as I guess there might be a quick fix, but I sadly couldn't spare lately as much time as I would have liked to. I'll try to have a look probably next week.

**Comment by feerrenrut** [comment]
@JulienCochuyt I'm in a similar position. My best guess is that this user has a different level of display scaling set in Windows display settings, and the explicit size of the tree control is pushing the controls of the screen. You could try scaling the sizes when creating the treeview.

I notice that it calls `mainsizer.Fit()`, I had some issues when using this incorrectly for `VisionProviderSubPanel_Wrapper`

**Comment by feerrenrut** [comment]
Ah, I can reproduce this now if I set my monitor resolution lower. I noticed in the image the the elements list dialog is taking up the whole screen. @lijianwei2019 if possible, you may have better luck with a higher screen resolution but using the "scaling" setting in Windows to make application bigger.

We have a hard-coded size for the tree-view in this dialog, I'm going to see if I can make it adapt to the screen resolution better.

**Comment by feerrenrut** [comment]
@lijianwei2019 Could you please test the [build of #10655](https://ci.appveyor.com/api/buildjobs/uhydc4ii863hlusm/artifacts/output%2Fnvda_snapshot_pr10738-19599%2C6a4d2ac6.exe) to confirm it fixes the issue for you?

**Comment by lijianwei2019** [comment]
@feerrenrut
ok.problem solved. Thank you for your work,

**Comment by lijianwei2019** [comment]
Resolution: 1366 x 768

**Comment by feerrenrut** [comment]
Fixed in commit c5ef3da1df3bc88d9854a922457fb367bc52f423 via PR #10738
